src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-relationships/GeneRelationships.scss

```diff
@@ -0,0 +1,35 @@
+@import 'src/styles/common';
+
+.fullWidthPanel {
+  width: 100%;
+  height: 100%;
+}
+
+.shrinkedPanel {
```

Comment:
1) ~shrinked~ -> shrunken
2) even better, shrunken -> narrow?
3) however, I don't think this is the effect Andrea was after:

![image](https://user-images.githubusercontent.com/6834224/76531595-56e0b000-646d-11ea-9708-f03cefa7baac.png)

I expect she would want the sidebar to overlap the panel as well, seeing that it started overlapping the rest of the content of entity viewer.
I have no idea whether it's possible to achieve this with CSS. Perhaps Andrea could give us the min width for the panel? I really don't understand how she wants us to draw it.

Reply:
I did speak to her about this and she preferred it to work this way.

I've renamed it to narrow.